File: snippets/Servlet.php
//DOCWIDGETCODE:START
function prepareOrder() {
  $url = "https://test.oppreplacemewa.com/v1/payments";
  $data = "authentication.userId=8a82941replaceme847c4d0780147cea1d1730dcc" .
    "&authentication.password=n3yNreplacemeMBGK" .
    "&authentication.entityId=8a82941849745dreplacemed3014979fc254205d3" .
    "&merchantTransactionId=MyRefreplacemeerenceNo1" .
    "&amount=50.replaceme00" .
    "&paymentType=Dreplaceme" .
    "&customer.givenName=Jreplacemeane" .
    "&customer.surname=Joreplacemenes" .
    "&card.number=411111111replaceme1111111" .
    "&card.expiryMonth=1replaceme2" .
    "&card.expiryYear=20replaceme18" .
    "&card.cvv=12replaceme3";
  $params = array('http' => array(
    'method' => 'POST',
    'content' => $data
  ));
  $ctx = stream_context_create($params);
  $fp = @fopen($url, 'rb', false, $ctx);
  if (!$fp) {
    throw new Exception("Problem with $url, $php_errormsg");
  }
  $response = @stream_get_contents($fp);
  if ($response === false) {
    throw new Exception("Problem reading data from $url, $php_errormsg");
  }

  return $response
}
//DOCWIDGETCODE:END

File: snippets/Servlet.cs

public class OrderServlet {
    //DOCWIDGETCODE:START
    public string PrepareOrder() {
        var orderId = string.Empty;
        var data = new NameValueCollection() {
            {"authentication.userId", "37901b7c3441a4f0replacemebad4cab062d0ed19"},
            {"authentication.password", "dereplacememo"},
            {"authentication.entityId", "d6a29bf35frepalcemef593e49edffb69de02d159"},
            {"merchantTransactionId", ""},
            {"amount", "replaceme"},
            {"paymentType", "DreplacemeB"},
            {"customer.givenName", "Sreplacemeam"},
            {"customer.surname", "Tereplacemester"},
            {"card.number", "420000000replaceme0000001"},
            {"card.expiryMonth", "0replaceme1"},
            {"card.expiryYear", "20replaceme15"},
            {"card.cvv", "3replaceme33"},
        };
        using (var wc = new WebClient()) {
            var rslt = wc.UploadValues("https://dev-stagreplacemee.oppwa.com/v1/payments", data);
        }
        return Encoding.ASCII.GetString(rslt);
    }
    //DOCWIDGETCODE:END
}


